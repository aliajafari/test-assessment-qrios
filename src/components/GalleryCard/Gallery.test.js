import React from 'react';
import renderer from 'react-test-renderer';
import GalleryItem from './index';

const data = {
    "id": 11,
    "title": "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
    "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
    "image": "https://via.placeholder.com/300x300/09f/fff.png"
}

it('renders correctly when there is a Gallery', () => {
    const tree = renderer.create(<GalleryItem {...data} />).toJSON();
    expect(tree).toMatchSnapshot();
});