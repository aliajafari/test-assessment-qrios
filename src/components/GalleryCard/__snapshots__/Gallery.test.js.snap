// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there is a Gallery 1`] = `
<div
  className="galleryItem"
>
  <div
    className="imageContainer"
    data-testid="gallery-item-image"
    onClick={[Function]}
    style={
      Object {
        "backgroundImage": "url(https://via.placeholder.com/300x300/09f/fff.png)",
      }
    }
  />
  <h2>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit
  </h2>
</div>
`;
